start-ocp-console.sh

```diff
@@ -11,7 +11,7 @@ INSTALLATION_NAMESPACE_MCE=`oc get multiclusterengine -A -o jsonpath='{.items[0]
 oc get secrets -n $INSTALLATION_NAMESPACE_MCE --field-selector type=kubernetes.io/service-account-token -o json | \
     jq '.items[0].data."ca.crt"' -r | python -m base64 -d > ocp-console/ca.crt
 
-CONSOLE_VERSION=${CONSOLE_VERSION:=4.15}
+CONSOLE_VERSION=${CONSOLE_VERSION:=4.16}
```

Comment:
QQ: Is this the new minimum supported OpenShift version?

Reply:
@Randy424 Yes. For the plugin minimum version I left it at 4.15, as 4.15 has RR6 and PF5 support, so there's no pressing reason to drop support for it, and sometimes customers might request a support exception.